tests/unit/test_contrib/test_repository.py

```diff
@@ -16,7 +16,7 @@
 
 
 def test_app_repository_signature_namespace() -> None:
-    app = Litestar([], on_app_init=[handlers.on_app_init])
+    app = Litestar([], on_app_init=[handlers.on_app_init], openapi_config=None)
```

Comment:
Well, yeh, but this was before I stripped out the work on the new openapi controller from this PR. That work added the `OpenaAPI` object to the signature namespace which would cause this test to fail as the test assumes that nothing else is adding anything to the namespace.

It can be removed.